pages/addEventListener_hook.coffee

```diff
@@ -0,0 +1,49 @@
+_addEventListener = Element::addEventListener
+
+elementsToRegister = []
+registrationElement = null
+
+EventTarget::addEventListener = (type, listener, useCapture) ->
+  eventTarget = if this in [document, window] then document.documentElement else this
+  if type == "click" and eventTarget instanceof Element
+    setTimeout (-> registerElementWithContentScripts eventTarget, "onclick"), 0
```

Comment:
This is injected into the page, so we have nothing except what's in this file available.